front/tests/005-operational-studies.spec.ts

```diff
@@ -1,33 +1,45 @@
 import { test, expect } from '@playwright/test';
+import { Project, Scenario, Study, RollingStock } from 'common/api/osrdEditoastApi';
 import { PlaywrightHomePage } from './pages/home-page-model';
 import PlaywrightRollingstockModalPage from './pages/rollingstock-modal-model';
 import PlaywrightMap from './pages/map-model';
-import VARIABLES from './assets/operationStudies/testVariables';
 import PATH_VARIABLES from './assets/operationStudies/testVariablesPaths';
 import PlaywrightScenarioPage from './pages/scenario-page-model';
-import { ProjectPage } from './pages/project-page-model';
-import { StudyPage } from './pages/study-page-model';
+import { findOneInResults, getApiRequest } from './assets/utils';
+
+let project: Project;
+let study: Study;
+let scenario: Scenario;
+let rollingStock: RollingStock;
+
+test.beforeAll(async () => {
+  const projects = await getApiRequest(`/api/projects/`);
+  project = findOneInResults(projects.results, 'project_test_e2e') as Project;
+
+  const studies = await getApiRequest(`/api/projects/${project.id}/studies/`);
+  study = findOneInResults(studies.results, 'study_test_e2e') as Study;
```

Comment:
Maybe we can create utilities function to get directly the projectId and studyId (same for scenarioId too), since you don't need the whole object
```suggestion
  projectId = await getProject();
  studyId = await getStudy(project.id);
```

Reply:
done